qt/python/mantidqt/mantidqt/widgets/sliceviewer/presenters/lineplots.py

```diff
@@ -342,7 +342,6 @@ def __init__(self, plotter: LinePlots, exporter: Any):
         self._selector = RectangleSelectorMtd(
             ax,
             self._on_rectangle_selected,
-            drawtype='box',
```

Comment:
I double-checked the api in 2.1.0 (for Ubuntu 18.04) and the default is `box` so removing this will have no effect there.